### ISSUE TITLE
Publish 0.1.0 to crates.io

### ISSUE THREAD

**Issue by haata** [bincode-org/bincode-core#14]
I'm trying to publish a crate dependent on this bincode-core.

The current api works well for my purposes (both x86_64 and arm-none-eabi).

**Comment by iamyulong** [comment]
@VictorKoenders thank you for creating this library. Do you have any plan to publish it to crates.io?

**Comment by VictorKoenders** [comment]
No, work is being done on making bincode itself able to run on `no_std` targets

**Comment by haata** [comment]
Excellent. It will be interesting to see how bincode will work without alloc (I need that for my project).

**Comment by haata** [comment]
I see https://github.com/bincode-org/bincode/releases/tag/v2.0.0-alpha.0 !
I'll give it a try to see if it's stable enough to use instead of bincode-core.

**Comment by VictorKoenders** [comment]
Are you already using bincode-core? we're considering archiving this repo in favor of bincode 2.0-alpha

**Comment by haata** [comment]
Yeah, I've been using bincode-core since Dec 2020 actually.
I really want to move to bincode 2.0-alpha asap though I'm still trying to figure out the differences.

The new derive  bincode::Encode macros failed pretty hard for me (was eating up the doc comment above the struct...) so I'll likely need to do some things manually.

**Comment by VictorKoenders** [comment]
Could you open up an issue on the [bincode](https://github.com/bincode-org/bincode) repo with the derive issue? I figured that'd break pretty hard for some systems.

Implementing a manual Decode/Encode is pretty simple now, an example is in the std test: https://github.com/bincode-org/bincode/blob/trunk/tests/std.rs#L17-L42

**Comment by haata** [comment]
Done, I've tried to isolate an enum and struct. It's not super clear what's going on without diving into the macros.

https://github.com/bincode-org/bincode/issues/427

(also, feel free to close this topic and archive this git repo)
